src/kinematics/DiffDriveKinematics.cpp

```diff
@@ -1,6 +1,10 @@
 #include "DiffDriveKinematics.h"
 
+#include "../Constants.h"
```

Comment:
This class shouldn't rely on constants. I think it's just because you use `Constants::MAX_WHEEL_VEL` below instead of `maxWheelVel`, which is an easy fix.